### ISSUE TITLE
Change `sample_color_bars` to `sample_highlights`?

### ISSUE THREAD

**Issue by mortonjt** [biocore/calour#60]
I feel that people will mix up the heatmap colorbars with the box indicating sample and feature categories.

What do you guys think about changing this parameter name to `sample_highlights` instead?  I think it would be semantically less confusing.

**Comment by amnona** [comment]
agree sample_color_bars is a bit confusing
vote for change to sample_highlights 👍

**Comment by RNAer** [comment]
yep. agree. @amnona , can we change it before you update your tutorial notebooks?
